dist/feature.js

```diff
@@ -0,0 +1,83 @@
+"use strict";
```

Comment:
gitignore `dist`

Reply:
OK, this confuses me – don't I need dist for NPM publishing, or that's never via the committed code?

Reply:
The CI should do the release after building it, right? That's how we do it in the DS